style.css

```diff
@@ -0,0 +1,344 @@
+}
+h3 {
+  font-size: 2.369rem;
+}
+h4 {
+  font-size: 1.777rem;
+}
+h5 {
+  font-size: 1.333rem;
+}
+small,
+.text_small {
+  font-size: 0.75rem;
+}
+
+header > div,
+footer > div {
+  padding: 2rem 2rem;
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  text-transform: uppercase;
+  font-weight: 600;
+  line-height: 2;
+  font-size: 0.875rem;
+  max-width: 35rem;
+  margin: 0 auto;
+}
+header a,
+footer a {
+  color: #cc99cd;
+  text-decoration: none;
+  max-height: 1.75rem;
+}
+header img,
+footer img {
+  height: 1.75rem;
+  -webkit-filter: invert(1);
+  filter: invert(1);
+  padding-right: 0.5rem;
+}
+footer {
+  margin-bottom: 6rem;
+}
+
+main {
+  max-width: 35rem;
+  margin: 0 auto;
+  background-color: black;
+}
+
+a:hover {
+  opacity: 0.8;
+}
+
+#output {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5em;
+  margin: 0.8em 0;
+  box-sizing: border-box;
+}
+
+pre {
+  overflow-x: scroll;
+}
+code,
+pre,
+code[class*="language-"],
+pre[class*="language-"] {
+  font-family: "Fira Code", Monaco, Menlo, Consolas, "Courier New", monospace;
+  position: relative;
+  font-size: 0.8rem;
+}
+pre[class*="language-"] {
+  padding: 0.5em 0.5em;
+  margin: 0em;
+  z-index: 2;
+}
+code,
+pre > code {
+  font-size: 0.8rem;
+  line-height: 1.42;
+  -webkit-text-size-adjust: 100%;
+}
+pre > code {
+  font-size: 0.8rem;
+  margin-left: 2.5%;
+  display: block;
+}
+
+:not(pre) > code[class*="language-"], pre[class*="language-"] {
+  background: #2d2d2d;
+  border-radius: 5px 5px 0px 0px;
+}
+pre[class*="language-"] + pre[class*="language-"].result {
+  background-color: #202020;
+  position: relative;
+  opacity: 0.9;
+  border-radius: 0px 0px 5px 5px;
+}
+
+pre[class*="language-"].result {
+  z-index: 1;
+}
+input[type="text"], textarea {
+  background-color: #2d2d2d;
+  outline: none;
+  color: #ccc;
+}
+
+#prompt-string {
+  font-family: "Nunito Sans", sans-serif;
+  font-weight: bold;
+  display: inline-block;
+  padding-right: 0.4em;
+}
+
+#input {
+  font-family: "Fira Code", Monaco, Menlo, Consolas, "Courier New", monospace;
+  background-color: #2d2d2d;
+  color: #d4d4d4;
+  border: none;
+  padding: 0.5em 1em;
+  border-radius: 5px 0px 0px 5px;
+  transition: background-color 200ms;
+  width: 20em;
+}
+
+#submit-input {
+  font-family: "Nunito Sans", sans-serif;
+  background-color: #3a3a3a;
+  color: #c0c0c0;
+  border: none;
+  padding: 0.5em 1.2em;
+  font-weight: bold;
+  border-radius: 0px 5px 5px 0px;
+  transition: background-color 200ms;
+}
+#submit-input:hover {
+  cursor: pointer;
+  background-color: #434343;
+}
+#submit-input:active {
+  background-color: #3a3a3a;
+}
+
+#clear-local-storage {
+  font-family: "Nunito Sans", sans-serif;
+  background-color: #2d2d2d;
+  color: #c0c0c0;
+  border: none;
+  padding: 0.5em 1.2em;
+  font-weight: bold;
+  border-radius: 5px;
+  transition: background-color 200ms;
+}
+#clear-local-storage:hover {
+  cursor: pointer;
+  background-color: #434343;
+}
+
+/*
```

Comment:
Want to take this out and add it back in later if you need it?

Reply:
Huh. No idea where that came from. I'll remove it.